### PR TITLE
Remove NetFx support libs

### PR DESCRIPTION
**Pull request by ericstj**
Now that we've shipped the NETStandard support package we will no longer build its content
from `master`.

It will be serviced from the `release/2.0.0` branch and the only netfx builds in `master` will
correspond to package content.

/cc @weshaggard

### CODE REVIEW

**Comment by ericstj** [comment]
test Windows x64 Debug Build

**Comment by ericstj** [comment]
test Tizen armel Debug Build
hitting https://github.com/dotnet/coreclr/issues/6676

**Comment by ericstj** [comment]
UWP test failed with 
```
11:23:37 EXEC : error PRI210: 0x80070020 - File move failed from 'C:\Users\DOTNET~1\AppData\Local\Temp\2ed4be99-49bf-497d-ba82-8a6414dda6a0' to 'D:\j\workspace\windows-TGrou---33cbf18b\bin\runtime\uap-Windows_NT-Debug-x64\resw'. [D:\j\workspace\windows-TGrou---33cbf18b\src\Microsoft.VisualBasic\tests\Microsoft.VisualBasic.Tests.csproj]
11:23:37 error PRI175 : 0x80070020 - MakePri failed with error : The process cannot access the file because it is being used by another process. [D:\j\workspace\windows-TGrou---33cbf18b\src\Microsoft.VisualBasic\tests\Microsoft.VisualBasic.Tests.csproj]
```

Looks to me like too generic of a path: `bin\runtime\uap-Windows_NT-Debug-x64\resw` @tarekgh @safern is this something you know about?

**Comment by ericstj** [comment]
Indeed, @safern looks like a race condition with this: https://github.com/dotnet/buildtools/commit/c45406f85a56c903d02eab582db455afa98dce3d#diff-425cb0dcf83ca9680e4a736c021dabe8R248.

test UWP CoreCLR x64 Debug Build

**Comment by ericstj** [comment]
test NETFX x86 Release Build
- event counter flaky test

**Comment by ericstj** [comment]
test UWP CoreCLR x64 Debug Build

**Comment by ericstj** [comment]
test UWP CoreCLR x64 Debug Build

**Comment by ericstj** [comment]
test NETFX x86 Release Build

File: src/System.Data.Common/ref/Configurations.props [hunk 1]
```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
-      netfx;
```
Comment (weshaggard) [inline src/System.Data.Common/ref/Configurations.props]: Can you please regenerate the VS projects/solutions?

Comment (ericstj) [reply src/System.Data.Common/ref/Configurations.props]: I'm planning to as a separate commit.  I want that in a place that is easy to re-run and not coupled with a bunch of manual project changes.

Comment (ericstj) [reply src/System.Data.Common/ref/Configurations.props]: I went ahead and did it in this PR, let's hope it doesn't reset with merge conflicts too much.

**Comment by ericstj** [comment]
test UWP CoreCLR x64 Debug Build
- hit #23614 
test NETFX x86 Release Build
- hit #22791

**Comment by ericstj** [comment]
test NETFX x86 Release Build

**Comment by ViktorHofer** [comment]
What about these? https://github.com/dotnet/corefx/blob/master/src/shims/manual/mscorlib.forwards.cs#L24

If we aren't building the shims for net461 anymore we can also remove these typeforwards in master.

**Comment by ericstj** [comment]
They need to remain because we're still building the facades for netcoreapp against the net641 targeting pack.

**Comment by ViktorHofer** [comment]
For how long will we continue doing that? I thought this was temporarily till we release the targeting pack.

**Comment by ericstj** [comment]
Until we update https://github.com/dotnet/corefx/pull/23611/files#diff-9db89a124f50762cf99552695cb75a50R17.  I am not doing that yet because it is still coupled to the meaning of `netfx`.  I split out remapping `netfx` to something newer because we're not ready to do that yet.

**Comment by ViktorHofer** [comment]
ok thanks

**Comment by ericstj** [comment]
@weshaggard can I get a review?

**Comment by ericstj** [comment]
>  used a common query to update all the project configurations

Yes, anything that set `IsNetFxNETStandard` got its netfx configuration removed.

**Pull request merged**